### ISSUE TITLE
completion lists should not be displayed after semicolon

### ISSUE THREAD

**Issue by mousetraps** [Microsoft/nodejstools#546]
```js
Foo.prototype.myFunction = function () {
    var self = this;// press enter here
```

Result:
```js
Foo.prototype.myFunction = function () {
    var self = this//<cursor>
```

Expected Result:
```js
Foo.prototype.myFunction = function () {
    var self = this;
    //<cursor>
```

**Comment by Earl-Brown** [comment]
Where do I up-vote this?

**Comment by mjbvz** [comment]
Fixed in ES6 Intellisense preview in NTVS 1.2 Alpha

![image](https://cloud.githubusercontent.com/assets/12821956/14267986/c904d2fc-fa8a-11e5-9c71-8ef840d7561c.png)
